Slider/Assets/Scripts/UI/Settings/FloatSettingUIElement.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FloatSettingUIElement : SettingUIElement<float>
+{
+
+}
```

Comment:
Unity serialization sucks x2

Reply:
this should probably be mentioned in a comment x2